src/components/header/Header.jsx

```diff
@@ -35,6 +39,13 @@ const Header = () => {
     sidenavDispatch({ type: TOGGLE_NAV });
   };
 
+  const applySearch = ({ key, target }) => {
+    if (key === "Enter") {
+      const filteredBySearch = filterByTitle(target.value, products);
+      filterDispatch({ type: PRODUCT_DATA, payload: filteredBySearch });
```

Comment:
you can pass payload as object for better readability and prophylaxis

Reply:
Ok, will try that